harfbuzz-sys/build.rs

```diff
@@ -11,14 +12,24 @@ fn main() {
         }
     }
 
-    assert!(Command::new("make")
-        .args(&["-R", "-f", "makefile.cargo", &format!("-j{}", env::var("NUM_JOBS").unwrap())])
-        .status()
-        .unwrap()
-        .success());
+    // On Windows, HarfBuzz configures atomics directly; otherwise,
+    // it needs assistance from configure to do so.  Just use the makefile
+    // build for now elsewhere.
+    let target = env::var("TARGET").unwrap();
+    if target.contains("windows") {
+        let dst = cmake::Config::new("harfbuzz").build();
+        println!("cargo:rustc-link-search=native={}", dst.display());
+        println!("cargo:rustc-link-lib=static=harfbuzz");
+    } else {
+        assert!(Command::new("make")
```

Comment:
Yep, we should do this as a followup.  Trying to convert this to a cross-platform cmakelist was actually painful, since Harfbuzz's configure generates a config.h with a number of important bits in it that I didn't want to track down right then.  Certainly possible to do, but for an infrequently-changing library like this I didn't think it was worth it right away.